### ISSUE TITLE
React + Redux + Typescript Chrome Extension Popup Script - Redux Devtools Shows "No Store Found" (Possible Solution?)

### ISSUE THREAD

**Issue by lbragile** [zalmoxisus/redux-devtools-extension#820]
Hi @zalmoxisus,

I asked the above question on [StackOverflow](https://stackoverflow.com/q/69699846/4298115)

One of the [responses](https://stackoverflow.com/a/69700412/4298115) suggested that when webpack applying 'uglification', then `window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__` in this [code](https://github.com/zalmoxisus/redux-devtools-extension/blob/master/npm-package/index.js#L6-L14) breaks.

Instead, using `window["__REDUX_DEVTOOLS_EXTENSION_COMPOSE__"]` should solve this issue.

Once I applied the above, I could finally see the store from the `redux` tab in the devtools window.
However, **my actions are still not being dispatched**. 

Is this due to my application being a chrome extension popup script?
Would using `remote-redux-devtool` fix the issue here?

**Comment by spirobel** [comment]
@lbragile did you get the actionCreators and tracing to work? It didn't work for me: https://github.com/reduxjs/redux-toolkit/issues/1815

**Comment by lbragile** [comment]
@spirobel Nope, I have the same issue as you described in your ticket - I just see a blank page in the `trace` tab when working with the remote redux devtools. However, on another project of mine, I can see the trace for the regular redux devtools. Thus, this is likely a bug with the remote version.

**Comment by spirobel** [comment]
true. So it is most likely a problem in the remote version. It seems like it is requesting the sources, but cant get them: https://imgur.com/a/BYzR0Nn
